.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,45 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "ramalama-python",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "."
+	},
+	//Needed for Podman in Podman 
+	"runArgs": [
+		"--userns=keep-id",
+		"--pid=host",
+		"--security-opt=label=disable",
+		"--security-opt=unmask=ALL",
+		"--device=/dev/fuse",
+		"--systemd=false",
+		"--device=/dev/dri",
```

Comment:
Will this fail if any of the referenced devices do not exist on a platform?

Reply:
Yeah... Generally --device fails if the device doesn't actually exist, sometimes I wish we had a --device-if-exists option in podman